velox/dwio/parquet/writer/Writer.h

```diff
@@ -108,6 +109,7 @@ struct WriterOptions : public dwio::common::WriterOptions {
   /// Timestamp time zone for Parquet write through Arrow bridge.
   std::optional<std::string> parquetWriteTimestampTimeZone;
   bool writeInt96AsTimestamp = false;
+  std::optional<arrow::ParquetDataPageVersion> parquetDataPageVersion;
```

Comment:
Velox should have its own enum and not borrow from Arrow.

Reply:
I must've misinterpreted one of your previous comments, and also felt that as we are no longer working in the HiveConnector, there was no need for an intermediary enum. I can add it back.